app/src/main/java/org/wikipedia/util/DateUtil.kt

```diff
@@ -72,8 +74,9 @@ object DateUtil {
         return getDateStringWithSkeletonPattern(date, "MMMM d")
     }
 
-    fun getMonthOnlyWithoutDayDateString(date: Date): String {
-        return getDateStringWithSkeletonPattern(date, "MMMM")
+    fun getMonthOnlyWithoutDayDateString(calendar: Calendar): String {
+        val localDateTime = LocalDateTime.ofInstant(calendar.toInstant(), ZoneId.systemDefault())
+        return localDateTime.month.getDisplayName(TextStyle.FULL, Locale.getDefault())
```

Comment:
Please explain the advantage of using these codes instead of the original ones — for example, efficiency or solving timezone issues.

Reply:
I updated the PR with some relevant changes, such as using [`MonthDay`](https://docs.oracle.com/javase/8/docs/api/java/time/MonthDay.html) in the date picker.